.github/workflows/terraform.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 The Sigstore Authors
+# SPDX-License-Identifier: Apache-2.0
+
+name: terraform-lint-validate
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  lint-and-validate:
+    name: terraform-lint-validate
+    runs-on: ubuntu-latest
+
+    if: github.repository == 'sigstore/sigstore-devops-tools'
+
+    steps:
+      - uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
+      - uses: hashicorp/setup-terraform@651471c36a6092792c552e8b1bef71e592b462d8 # v3.1.1
```

Comment:
should we use the same version as in the deploy step?

Reply:
just waiting on this to LGTM

Reply:
ahh sorry
missed that

Reply:
done